src/components/selectionBoxLayer.ts

```diff
@@ -183,9 +225,21 @@ export module Components {
      *
      * Returns an undefined array if the edges are not backed by a scale.
```

Comment:
Then it returns "the data values backing the left and right edges of the box."

The statement is mainly to satisfy the edge case.